libraries/APM_Control/AR_AttitudeControl.cpp

```diff
@@ -207,7 +207,7 @@ float AR_AttitudeControl::get_steering_out_rate(float desired_rate, bool skid_st
     // calculate dt
     const uint32_t now = AP_HAL::millis();
     float dt = (now - _steer_turn_last_ms) / 1000.0f;
-    if ((_steer_turn_last_ms == 0) || (dt > AR_ATTCONTROL_TIMEOUT_MS)) {
+    if ((_steer_turn_last_ms == 0) || (dt > (AR_ATTCONTROL_TIMEOUT_MS / 1000.0f))) {
```

Comment:
Maybe change the define to be in seconds instead?

Reply:
I guess the timeout check is done in two slightly different ways - one method uses seconds, the other milliseconds... so consolidating the method might be a good thing to do..